README.md

```
@@ -5,3 +5,48 @@
+
+### 2. 환경변수 설정
+
+`.env` 파일 생성:
+
+```bash
+SECRET_KEY=your-secret-key-here
+PASSWORD_PEPPER=your-pepper-key-here
+```
+
+키 생성 방법:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)